manuscript/chapter3.txt

```diff
@@ -7,29 +7,29 @@ C> By [John Haynes](https://www.youtube.com/channel/UCJVXolxwB4x3EsBAzSACCTg) |
 
-A discussion of password hashes would not be complete without mentioning salted passwords. The salt for a password is additional data that is added to the password before the hash algorithm is applied to complicate the guessing of the password. Therefore, the salt would have to be known and applied to each potential guess otherwise the hash would be incorrect even if the correct password was guessed. The salt can be generated in several different ways and can be static or dynamic depending on developer choice. Unfortunately, Windows does not salt the NTLM password hashes that it generates so they are vulnerable to attack.
+Some hashing algorithms are more complicated and slower to crack, security is prioritized over decryption speed. To make  password cracking more difficult, the hashing algorithm can also introduce extra data, known as "salt," increasing processing time and foiling unsophisticated attempts. The correct salt must be applied to each guess, or the hash of even a correct password will not match the original. There are several ways to generate salt, and it can be static or dynamic.
 
-As was just mentioned, Windows stores password hashes in NTLM format. This is unfortunately a very weak form of encryption as it is the equivalent of MD4 encryption. The VTech company was compromised in 2015 by a SQL injection attack and when the password hashes were analyzed they were determined to be encrypted with MD5. MD5 is considered to be a weak form of encryption and some do not consider it to even be encryption as it is so weak. Windows uses even weaker encryption for its passwords, and those passwords are not even salted to compensate for the weak encryption! Windows has upgraded to NTLMv1 and NTLMv2 for some uses, but those are still weak by most encryption standards. Even more concerning is these NTLM hashes of user passwords are transmitted over the network for authentication between computers (Patton, 2022). This is one of the most common passwords that users will use and can be extracted by several methods, including packet sniffing. It is also nearly guaranteed to not be generated by a password manager as the user has to physically enter the password into the keyboard.
+Notably, Windows does not salt its New Technology LAN Manager (NTLM) password hash, rendering it very vulnerable to attack. Further, the hashing algorithm is very weak, and the hashed passwords are transmitted over networks, making them vulnerable to packet-sniffing attacks (Patton, 2022). These passwords are typically simpler than ones generated by password managers since the user must physically type them to log in to the computer.
```

Comment:
A little additional info, either what is in bold or something similar.

"Notably, Windows does not salt its New Technology LAN Manager (NTLM) password hash, rendering it very vulnerable to attack. **This is the password used to log into the Windows machine.** Further, the hashing algorithm is very weak, and the hashed passwords are transmitted over networks, making them vulnerable to packet-sniffing attacks (Patton, 2022)."